### PR TITLE
Update discord_integration.py

### PR DESCRIPTION
**Pull request by NekoTony**
* **Please check if the PR fulfills these requirements**
- [ ] Tests have been conducted on the PR
- [ ] Docs have been added / updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
A fix for the discord_integration.py file so it works.

* **What is the current behavior?** (You can also link to an open issue here)
Discord_integration.py is out of date

* **What is the new behavior (if this is a feature change)?**
Discord_integration.py is now fixed to work with latest version of d.py

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

**Comment by IAmTomahawkx** [comment]
youve included your credentials in this pull request. I would suggest [deauthorizing](https://dev.twitch.tv/docs/authentication#revoking-access-tokens) that token

File: examples/discord_integration.py [hunk 1]
```diff
@@ -24,40 +24,43 @@
...
-
-        # Start the Twitch Bot
-        self.loop.create_task(self.start())
+        self.bot = commands.Bot(
+            # set up the bot
+            irc_token='',
+            client_id='',
+            nick='ArthritisAC',
+            prefix='--',
+            initial_channels=['#ArthritisAC']
```
Comment (IAmTomahawkx) [inline examples/discord_integration.py:40]: `initial_channels=['...']`

File: examples/discord_integration.py [hunk 2]
```diff
@@ -24,40 +24,43 @@
...
         self.discord_bot = bot
-        super().__init__(irc_token='...', nick='...', prefix='!', initial_channels=['...'])
-
-        # Start the Twitch Bot
-        self.loop.create_task(self.start())
+        self.bot = commands.Bot(
+            # set up the bot
+            irc_token='',
+            client_id='',
+            nick='ArthritisAC',
```
Comment (IAmTomahawkx) [inline examples/discord_integration.py:38]: `nick=''`

**Comment by NekoTony** [comment]
Fixed and it's revoked, thanks.

File: examples/discord_integration.py [hunk 3]
```diff
@@ -24,40 +24,43 @@
 DEALINGS IN THE SOFTWARE.
 """
 
+import discord
```
Comment (IAmTomahawkx) [inline examples/discord_integration.py:27]: redundant import

**Review by IAmTomahawkx — CHANGES_REQUESTED** [review]

**Comment by IAmTomahawkx** [comment]
outdated

**Pull request closed without merging**